exercise_002_cluster_base/src/main/resources/application.conf

```diff
@@ -36,6 +36,13 @@ akka {
                   "akka://pi-"${cluster-node-configuration.cluster-id}"-system@"${cluster-node-configuration.seed-node-2}":2550",
                   "akka://pi-"${cluster-node-configuration.cluster-id}"-system@"${cluster-node-configuration.seed-node-3}":2550",
                   "akka://pi-"${cluster-node-configuration.cluster-id}"-system@"${cluster-node-configuration.seed-node-4}":2550"]
+
+    gossip-interval = 2s
+    gossip-time-to-live = 4s
```

Comment:
@manuelbernhardt 
Note: It's probably a good idea to add a comment stating that this parameter tweaking is done to slow down the convergence process for demo purposes and that this shouldn't be done unless one knows very well why. WDYT?